src/components/contentLibrary/contentLibrary.component.tsx

```diff
@@ -28,8 +54,20 @@ export const ContentLibrary = () => {
     };
   }, [activePanelObject, objectListingWidth]);
 
+  useEffect(() => {
```

Comment:
I'm concerned that this might trigger a rerender each time the mouse moves because you're saving it to state

Reply:
switched to `mousedown`, same issue, fewer times....

Reply:
useRef nice